swap_meet/vendor.py

```diff
@@ -1,2 +1,121 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
```

Comment:
Great job working with mutable default parameters! Great use of the ternary as well!